src/node_modules/_file-explorer.js

```diff
@@ -0,0 +1,292 @@
+var setval = require('setval')
```

Comment:
This is something I would like to discuss during the next meeting. Until then, please keep to the current style.